controllers/controllers_test.go

```diff
@@ -118,4 +119,38 @@ var _ = Describe("Identity Controller", func() {
 			Expect(body.SmartMangement.IsEntitled).To(Equal(false), "smart_management.is_entitled expected to be false")
 		})
 	})
+
+	Context("When the account number is -1 or '' ", func() {
+		var fakeResponse SubscriptionsResponse
+
+		BeforeEach(func() {
+			fakeResponse = SubscriptionsResponse{
+				StatusCode: 200,
+				Data:       []string{"foo", "bar"},
+				CacheHit:   false,
+			}
+		})
+
+		It("should give back a valid EntitlementsResponse with insights false", func() {
+			// testing with account number "-1"
+			rr, body, _ := testRequest("GET", "/", "-1", DEFAULT_ORG_ID, fakeGetSubscriptions(DEFAULT_ORG_ID, fakeResponse))
+			expectPass(rr.Result())
+			Expect(body.Insights.IsEntitled).To(Equal(false), "insights.is_entitled expected to be false")
```

Comment:
Why does the first `To()` clause have an explanation string attached to it, but the `To()` clauses in the following lines don't?

Reply:
@kruai I think this is because we are expecting the entitlement of Insights to be false and everything else to be true. This should be fine. :slightly_smiling_face: